extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -23,6 +23,72 @@ limitations under the License.
   </tr>
   <tr>
     <td width="40%"><strong>Availability</strong></td>
-    <td><div>Experimental</div><div>Work in progress.</div></td>
+    <td><div><a href="https://www.ampproject.org/docs/reference/experimental.html">Experimental</a>; no validations yet.</div><div>Work in progress.</div></td>
```

Comment:
Looks like the changes are already on goldmine :) ill have to file bugs for some minor changes.